sdcm/cluster.py

```diff
@@ -3372,6 +3374,21 @@ def kill_gemini_thread(self):
                 if kill_result.exit_status != 0:
                     self.log.error('Terminate gemini on node %s:\n%s', loader, kill_result)
 
+    def create_swap_file(self):
+        swap_size = self.params.get("loader_swap_size", default=None)
+        if not swap_size:
+            self.log.info("Swap file is not configured")
+            return
+        commands = dedent("""sudo /bin/dd if=/dev/zero of=/var/swap.1 bs=1M count={}
+                          sudo /sbin/mkswap /var/swap.1
+                          sudo chmod 600 /var/swap.1
+                          sudo /sbin/swapon /var/swap.1""".format(swap_size))
```

Comment:
how about adding a grep statement to make sure if swap file is using?
grep '/var/swap.1' /proc/swaps

Reply:
agree

Reply:
Thanks. added